src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbCollectionChecker.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Security.Authentication;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Exceptions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using Polly;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Mongo
+{
+    public class MongoDbCollectionChecker
```

Comment:
I understand the check but not sure I understand the purpose of the check. The wireup stops jobs from running which sounds efficient but what about QA web and employer web?

Previously we were checking the collection everytime an interaction was going to be made which I agree looks excessive. 

The reason for it was because `applicationReviews` collection was created in PROD without it being declared in the Cosmos ARM config template so it didn't have the correct setup. The previous check was to ensure no V2 component application would create the collection automatically.

Reply:
Yes please

Reply:
👍